### PR TITLE
add option to pass style to container

### PR DESCRIPTION
**Pull request by adamszeptycki**

### CODE REVIEW

File: src/ReactCardFlip.jsx [hunk 1]
```diff
@@ -53,7 +54,9 @@ class ReactCardFlip extends React.Component {
       },
       flipper: {
         position: 'relative',
-        transformStyle: 'preserve-3d'
+        transformStyle: 'preserve-3d',
+        width: '100%',
```
Comment (AaronCCWong) [inline src/ReactCardFlip.jsx]: Is there any reason for setting width and height to 100%?

Comment (RichMatthews) [reply src/ReactCardFlip.jsx]: I really need this too, would be good to have the option to apply additional styles as I need it taking 100% of my container

Comment (adamszeptycki) [reply src/ReactCardFlip.jsx]: The reason to set it to 100% was that I can have enclosing div take care of sizing.
Right now styling is a little bit of a hassle.

**Comment by AaronCCWong** [comment]
Thanks for adding this! Sorry for taking so long to get it merged.

**Pull request merged**